PubNub/Data/Service Objects/PNSubscriberResults.h

```diff
@@ -27,6 +27,8 @@
  */
 @property (nonatomic, readonly, strong) NSNumber *timetoken;
 
+@property (nonatomic, readonly, strong) NSNumber *region;
```

Comment:
@jzucker2 Is there requirements in specs to expose region? As I remember, it is completely internal information. Only userMeta should be exposed here (if docs not changed since I viewed them last time).